### ISSUE TITLE
scipy.weave depreacted

### ISSUE THREAD

**Issue by hkennyv** [ELELAB/pyinteraph#5]
It looks like weave was [deprecated](https://docs.scipy.org/doc/scipy/reference/release.0.15.0.html#deprecated-features) in scipy and moved to its own project.

```
  File "/Users/khuynh/stackoverflow/pyi/venv/lib/python2.7/site-packages/MDAnalysis/analysis/distances.py", line 31, in <module>
    from scipy import weave
ImportError: cannot import name weave
```

I was able to resolve this quickly by 

1. installing weave via `pip install weave`
2. editing the file in scipy at `venv/lib/python2.7/site-packages/MDAnalysis/analysis/distances.py`

and changing:

```python
import numpy
from scipy import sparse
from scipy import weave
from scipy.weave import converters
```

to:

```python
import numpy
from scipy import sparse
#  from scipy import weave
#  from scipy.weave import converters

import weave
from weave import converters
```

is there a more elegant way to do this? i don't think modifying the source of scipy is the most graceful way.. open to suggestions? :)

I've also written a stackoverflow answer for this [here](https://stackoverflow.com/questions/59829125/problem-with-python-undefined-symbol-sqmi?noredirect=1#comment105796650_59829125).

**Comment by mtiberti** [comment]
Hi,
thanks for your efforts! Alternatively, you can simply comment out the offending line in libinteract (line 2), I just realised it's a stray import and imported function is not actually used in the code. I'll add this to the list of fixes - we are planning on releasing a Py3 compatible version which uses the latest MDAnalysis as well and will make things much simpler - stay tuned!

**Comment by hkennyv** [comment]
Thank you and thanks for your work too, I look forward to seeing the py3 release! Is it worth submitting a PR to fix this in the 2.7 version or is development here halted?

**Comment by elenapapaleo** [comment]
We have now the py3 release here: https://github.com/ELELAB/pyinteraph2
